GiEnJul/Startup.cs

```diff
@@ -28,8 +33,26 @@ public void ConfigureServices(IServiceCollection services)
             {
                 configuration.RootPath = "ClientApp/build";
             });
+            string domain = $"https://{Configuration["Auth0:Domain"]}/";
+            services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+            })
+            .AddJwtBearer(options =>
+            {
+                var section = Configuration.GetSection("Auth0");
+                options.Authority = domain;
+                options.Audience = section.GetValue<string>(_env.IsDevelopment() ? "LocalAudience" : "AzureAudience");
+            }
+            );
+            services.AddAuthorization(options =>
+            {
+                options.AddPolicy("read:givers", policy => policy.Requirements.Add(new HasScopeRequirement("read:givers", domain)));
```

Comment:
Does this allow us to use [Authorize("read:givers")] in controllers?

Reply:
It is now moved to the GiEnJul.Auth namespace and you can now write something like this:  [Authorize(Policy = "ReadGivers")]  in the controllers